src/components/swap/SwapModalFooter.tsx

```diff
@@ -111,8 +161,12 @@ export default function SwapModalFooter({
   swapQuoteReceivedDate,
   fiatValueInput,
   fiatValueOutput,
+  shouldLogModalCloseEvent,
+  setShouldLogModalCloseEvent,
+  showAcceptChanges,
+  onAcceptChanges,
 }: {
-  trade: InterfaceTrade<Currency, Currency, TradeType>
+  trade: InterfaceTrade<Currency, Currency, TradeType> | undefined
```

Comment:
shouldn't trade always be defined by time you open the review modal?

Reply:
+1. Let's make sure we do not pass undefined/nulls as they're confusing when it comes to `trade`.